pkg/oc/cli/admin/network/isolate_projects.go

```diff
@@ -35,7 +35,7 @@ type IsolateOptions struct {
 	Options *ProjectOptions
 }
 
-func NewCmdIsolateProjectsNetwork(commandName, fullName string, f kcmdutil.Factory, out io.Writer) *cobra.Command {
+func NewCmdIsolateProjectsNetwork(commandName, fullName string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
 	opts := &ProjectOptions{}
 	isolateOp := &IsolateOptions{Options: opts}
```

Comment:
We should have wrapper methods that call the embedded ProjectOptions Complete and Validate methods